scss/inc/_navigatorFizzy.scss

```diff
@@ -0,0 +1,331 @@
+    }
+
+    /* reset parent style */
+    button [class*=' icon-'] {
+        padding: 0;
+    }
+
+    /* reset parent styles */
+    .qti-navigator-filters.key-navigation-group.focusin.focusin {
+        background: $uiReviewPanelBg;
+    }
+    .qti-navigator-filters {
+        height: calc(100% - #{$headerHeight});
+        margin: 0 $panelPadding; /* shows focusing border around tree items */
+    }
+
+    /* header */
+    & > .qti-navigator-label {
+        display: flex;
+        justify-content: space-between;
+        margin: 0 $panelPadding;
+        border-bottom: 1px solid #ccc;
+        min-height: $headerHeight;
+        align-items: center;
+
+        .qti-navigator-text {
+            @include font-size(20);
+            font-weight: bold;
+        }
+
+        .icon-close {
+            @include font-size(28);
+            line-height: $lineHeight;
+            color: $uiReviewPanelTextDefault;
+
+            &:hover {
+                color: $uiReviewPanelTextDefault;
+                text-decoration: none;
+                cursor: pointer;
+            }
+        }
+    }
+
+    /* tree */
+    .qti-navigator-tree {
+        background: none;
+
+        span.qti-navigator-label {
+            margin: 13px 0 10px 0;
+            display: block;
+            @include font-size(15);
+        }
+    }
+
+    .qti-navigator-icon {
+        padding: 0;
+        top: 0;
+        left: 0;
+    }
+
+
+    & .icon-flagged::before {
+        @include icon-bookmark;
+    }
+
+    & .icon-info::before {
+        @include icon-info-bare;
+    }
+    & .icon-unseen {
+        display: none;
+    }
+    & .icon-viewed {
+        display: none;
+    }
+
+    .qti-navigator-item {
+        height: $hitboxSize;
+        width: $hitboxSize;
+        display: flex;
+        justify-content: center; /* aligning left - cut focusing board for keyboard interacting */
+        align-items: center;
+        /* reset parent styles */
+        padding: 0 !important; /* instead of rewriting 6-classes selector on focusing */
+        border: none !important; /* instead of rewriting 6-classes selector on focusing */
+
+        /* reset global styles */
+        &.disabled {
+            background-color: unset !important;
+            color: $uiReviewPanelTextDisabled !important;
+            border-color: $uiReviewPanelTextDisabled !important;
+            opacity: 1 !important;
+            text-shadow: none !important;
+        }
+
+        &.disabled .step {
+            border-width: $borderMedium;
+            color: $uiReviewPanelTextDisabled;
+            border-color: $uiReviewPanelTextDisabled;
+            opacity: 1;
+            text-shadow: none;
+        }
+
+        /* remove number from item button if item was marked for review */
+        &.flagged span.step-label {
+            display: none;
+        }
+        /* remove number from item button if item is informational */
+        &.info span.step-label {
+            display: none;
```

Comment:
Double-check: In NextGen we don't increase item index (and total) for informational items. Is it ok that here we don't? I'm fine with it, just confirming. 

![Screenshot_17](https://user-images.githubusercontent.com/38751932/141342089-5c027b43-f301-4951-89fd-b2f3df5694c6.png)

Reply:
Yeah. This is in progress. I hoped i finish it faster)

Reply:
I've added solution here